mayastor/src/subsys/nvmf/subsystem.rs

```diff
@@ -269,6 +270,30 @@ impl NvmfSubsystem {
         Ok(())
     }
 
+    /// set controller ID range
+    pub fn set_cntlid_range(
+        &self,
+        cntlid_min: u16,
+        cntlid_max: u16,
+    ) -> Result<(), Error> {
+        unsafe {
+            spdk_nvmf_subsystem_set_cntlid_range(
```

Comment:
Is this function part of standard SPDK API ? Where it resides ? (I wasn't able to find it via https://spdk.io/doc/)

Reply:
It's not, the PR for that wasn't ready at the time, it is now openebs/spdk#12.

Reply:
I've dropped the `spdk_` prefix.